src/library-authoring/EmptyStates.tsx

```diff
@@ -7,16 +8,22 @@ import { Add } from '@openedx/paragon/icons';
 import { ClearFiltersButton } from '../search-manager';
 import messages from './messages';
 import { LibraryContext } from './common/context';
+import { useContentLibrary } from './data/apiHooks';
 
 export const NoComponents = () => {
   const { openAddContentSidebar } = useContext(LibraryContext);
+  const { libraryId } = useParams();
+  const { data: libraryData } = useContentLibrary(libraryId);
+  const canEditLibrary = libraryData?.canEditLibrary ?? false;
 
   return (
-    <Stack direction="horizontal" gap={3} className="mt-6 justify-content-center">
+    <Stack direction="horizontal" gap={3} className="mt-6 just  ify-content-center">
```

Comment:
```suggestion
    <Stack direction="horizontal" gap={3} className="mt-6 justify-content-center">
```

Reply:
Ohh, It's my keyboard's fault :laughing: 

Fixed: https://github.com/openedx/frontend-app-course-authoring/pull/1198/commits/79154a4fba0acd0f4cab2bddff728f1a891ae43e